### PR TITLE
Upgrade Ember to v3.7

### PR DESCRIPTION
**Pull request by Haz901**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -46,6 +46,7 @@ jobs:
     - stage: "Additional Tests"
       env: EMBER_TRY_SCENARIO=ember-lts-2.16
     - env: EMBER_TRY_SCENARIO=ember-lts-2.18
+    - env: EMBER_TRY_SCENARIO=ember-lts-3.4
```
Comment (cbroeren) [inline .travis.yml]: Can the 2.16 be removed now, or is it still in the blueprint too?

File: LICENSE.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2016
+Copyright (c) 2019
```
Comment (cbroeren) [inline LICENSE.md]: Hihi 👍

File: tests/integration/components/model-picker-test.js [hunk 3]
```diff
@@ -246,23 +246,27 @@ module('Integration | Component | {{model-picker}}', function(hooks) {
 
     let $items = $('.ember-power-select-dropdown li');
 
-    assert.equal($items[0].innerText, 'bar',        'reloaded first item sorted');
-    assert.equal($items[1].innerText, 'bar second', 'reloaded second item sorted');
-    assert.equal($items[2].innerText, 'chad',       'reloaded third item sorted');
-    assert.equal($items[3].innerText, 'dave',       'reloaded fourth item sorted');
+    assert.equal($items[0].textContent.trim(), 'bar',
+      'reloaded first item sorted');
```
Comment (cbroeren) [inline tests/integration/components/model-picker-test.js]: Maybe add a function to keep the alignment / readability:
```js
let getText = (item) => item.textContent.trim();

assert.equal(getText($items[0]), 'bar',        'reloaded first item sorted');
assert.equal(getText($items[1]), 'bar second', 'reloaded second item sorted');
assert.equal(getText($items[2]), 'chad',       'reloaded third item sorted');
assert.equal(getText($items[3]), 'dave',       'reloaded fourth item sorted');
```

File: .travis.yml (hunk 1 above)
Comment (Haz901) [reply .travis.yml]: Could be removed, it is recommended to test at least the current LTS (3.4) and the previous one (2.18), keeping 2.16 in is optional

**Pull request merged**